libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -2129,7 +2129,9 @@ bool ExpressionCompiler::cleanupNeededForOp(Type::Category _type, Token::Value _
 {
 	if (Token::isCompareOp(_op) || Token::isShiftOp(_op))
 		return true;
-	else if (_type == Type::Category::Integer && (_op == Token::Div || _op == Token::Mod))
+	else if (_type == Type::Category::Integer && (_op == Token::Div || _op == Token::Mod || _op == Token::Exp))
+		// We need cleanup for EXP because 0**0 == 1, but 0**0x100 == 0
+		// It would suffice to clean the oponent, though.
```

Comment:
Spellchecker found `oponent  ==> opponent`.
Is it a typo though? (exponent)

Reply:
Good old spelly!